docs/docs/test-runner/browser-launchers/overview.md

```diff
@@ -46,3 +46,19 @@ export default {
   browsers: [chromeLauncher({ launchOptions: { args: ['--no-sandbox'] } })],
 };
 ```
+
+
+### Configuring browser timezone
+```js 
+import { playwrightLauncher } from '@web/test-runner-playwright';
```

Comment:
Please consider moving this code snippet to Playwright launcher documentation page.

Reply:
right, I was assumed the configuration is applicable to all browser luncher

Reply:
It's probably a good idea to change `Configuring browser timezone` to `Providing browser context`.
Also, please provide some text description and not only code snippet.